Exercism/DifferenceOfSquares/DifferenceOfSquaresTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using Xunit;
+
+namespace Exercism.DifferenceOfSquaresTests
+{
+    [Trait("Category", "DifferenceOfSquares")]
+    public class DifferenceOfSquaresTests
```

Comment:
Don't need to change it now, but be aware [that this can all be done with an xunit `Theory`](https://xunit.github.io/docs/getting-started-desktop.html#write-first-theory)